### PR TITLE
【论文复现营】torch.as_tensor(data, dtype=None, device=None) ->  paddle.to_tensor(d…

### PR DESCRIPTION
**Pull request by wyxogo**
torch.as_tensor(data, dtype=None, device=None)
->
paddle.to_tensor(data,dtype=None, place=None, stop_gradient=True)

### CODE REVIEW

File: x2paddle/project_convertor/pytorch/api_mapper/totensor.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    def process_attrs(self):
+        rename_key(self.kwargs, "device", "place")
+    def run(self):
+        if self.rename_func_name("paddle.to_tensor"):
+            return [], generate_api_code(self.func_name, self.args, self.kwargs), []
+        else:
+            # 作用：将paddle与pytorch不同的可变参数替换成字典参数，并生成相应代码
+            self.convert_args2kwargs()
+            return self.convert_to_paddle()
```
Comment (SunAhong1993) [inline x2paddle/project_convertor/pytorch/api_mapper/totensor.py]: 建议将此处代码放置于x2paddle/project_convertor/pytorch/api_mapper/ops.py

File: x2paddle/project_convertor/pytorch/mapper.py [hunk 2]
```diff
@@ -24,7 +24,8 @@
     "torch.optim.lr_scheduler.MultiStepLR":
     ["paddle.optimizer.lr.MultiStepDecay", LRScheculerMapper],
     "torch.optim.Adam": ["x2paddle.torch2paddle.Adam", None],
-    "torch.optim.SGD": ["x2paddle.torch2paddle.Momentum", None]
+    "torch.optim.SGD": ["x2paddle.torch2paddle.Momentum", None],
+    "torch.as_tensor": ["paddle.to_tensor", ToTensor]
```
Comment (SunAhong1993) [inline x2paddle/project_convertor/pytorch/mapper.py]: 建议将此行代码放置于`API_MAPPER`这个字典中

File: x2paddle/project_convertor/pytorch/api_mapper/totensor.py (hunk 1 above)
Comment (wyxogo) [reply x2paddle/project_convertor/pytorch/api_mapper/totensor.py]: > 建议将此处代码放置于x2paddle/project_convertor/pytorch/api_mapper/ops.py

已做修改

File: x2paddle/project_convertor/pytorch/mapper.py [hunk 3]
```diff
@@ -292,6 +293,7 @@
     "torch.mul": ["paddle.multiply", TwoMathMapper],
     "torch.einsum": ["paddlenlp.ops.einsum ", None],
     "torch.linspace": ["paddle.linspace", LinspaceMapper],
+    "torch.as_tensor": ["paddle.to_tensor", ToTensor],
```
Comment (SunAhong1993) [inline x2paddle/project_convertor/pytorch/mapper.py]: 建议`ToTensor `改为`ToTensorMapper`

File: x2paddle/project_convertor/pytorch/api_mapper/ops.py [hunk 4]
```diff
@@ -538,3 +538,20 @@ def run(self):
                 out2 = "{}.requires_grad_({})".format(
                     out2, self.useful_attrs["requires_grad"])
             return out1, out2, out3
+
+class ToTensor(Mapper):
```
Comment (SunAhong1993) [inline x2paddle/project_convertor/pytorch/api_mapper/ops.py]: 建议`ToTensor `改为`ToTensorMapper`

File: x2paddle/project_convertor/pytorch/mapper.py (hunk 3 above)
Comment (wyxogo) [reply x2paddle/project_convertor/pytorch/mapper.py]: > 建议`ToTensor `改为`ToTensorMapper`

已做修改

**Comment by wyxogo** [comment]
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**